RandomData.Api/NumberGenerators/Dtos/NumberParameters.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace RandomData.Api.NumberGenerators.Dto
+{
+    public class NumberParameters
+    {
+        private const string DefaultAllowedDigits = "1234567890";
```

Comment:
If I were you I would also add options such as:
- min value
- max value
- base (10, 2, 4, 8 or more)

Reply:
I will consider adding it in a separate route
min and max value won't work well with length and allowed digits parameters, it will make validation and logic very complicated

Reply:
Well currently it works almost identical to string endpoint with just numbers specified so that's why I would focus more on stuff that cannot be achieved elsewhere

Reply:
I kept to what we determined during our meeting - max and min length for a number endpoint